ansible/roles/openshift_udp_limits/files/iptables_udp.conf

```diff
@@ -0,0 +1,2 @@
+# Ansible-managed
+if $syslogfacility-text == 'kern' and $msg startswith 'UDP outlog: ' then -/var/log/iptables.log
```

Comment:
we still want the `& stop` or the logs will end up in `/var/log/messages` as well as `/var/log/iptables.log`.